tests/E2ETests/test_pytom_FullPipeline.py

```diff
@@ -543,7 +613,7 @@ def test_18_GLocal_Reduced_Binned_GPU(self):
 
         if not os.path.exists(outdir): os.mkdir(outdir)
 
-        cmd = f'cd {self.projectname}/05_Subtomogram_Analysis; mpiexec -n 5 pytom {self.pytomDir}/bin/GLocalJob.py '
+        cmd = f'cd {self.projectname}/05_Subtomogram_Analysis; mpiexec -n 2 pytom {self.pytomDir}/bin/GLocalJob.py '
```

Comment:
Shouldn't this be fixed as well?
```suggestion
        cmd = f'cd {self.projectname}/05_Subtomogram_Analysis; mpiexec -n 2 GLocalJob.py '
```